torch/onnx/_internal/exporter.py

```diff
@@ -698,26 +696,37 @@ def __call__(
         Args:
             args: The positional inputs to the model.
             kwargs: The keyword inputs to the model.
-            model_with_state_dict: The PyTorch model to fetch state from.
-                Required when :func:`enable_fake_mode` is used to extract real initializers as needed by the ONNX graph.
+            exported_program: Required when :func:`enable_fake_mode` is used to extract real initializers as
```

Comment:
how about the non-exportedprogram path?

Reply:
non-exportedprogram path doesn't need to pass in model to run.